source/engine/graphics/api/vulkan/core/Instance.cpp

```diff
@@ -8,7 +8,7 @@
 
 namespace
 {
-constexpr auto g_validationLayers = std::array<const char*, 1>{ "VK_LAYER_KHRONOS_validation" };
+constexpr auto g_validationLayers = std::array<const char*, 2>{ "VK_LAYER_KHRONOS_validation", "VK_LAYER_LUNARG_api_dump" };
```

Comment:
Neat, does this just automatically look for the file in the current directory? Is it well behaved if we don't have the file, but still want validation layers?

Reply:
Yes it does look in the current dir. It more or less is well behaved (it defaults to stdout, so it's a lot of text on the console, though only when you have validation layers enabled which is already signing up for text in stdout)